coast/config/config_parser.py

```diff
@@ -0,0 +1,84 @@
+"""Config parser."""
+import json
+from pathlib import Path
+from typing import Union
+
+from .config_structure import (
+    ConfigTypes,
+    ConfigKeys,
+    GriddedConfig,
+    IndexedConfig,
+    Dataset,
+    Domain
+)
+
+class ConfigParser():
+    """Class for parsing gridded and indexed configuration files."""
+    def __init__(self, json_path: Union[Path, str]):
+        """Config parser constructor.
+        
+        Args:
+            json_path (Union[Path, str]): path to json config file.
+        """
+        with open(json_path, 'r') as j:
```

Comment:
We can drop the with block and use `json.loads(Path(json_path).read_text())`.

Reply:
Will notify Rob.